code/upstream-integration-tests/src/androidTest/java/com/adobe/marketing/mobile/edge/integration/UpstreamIntegrationTests.kt

```diff
@@ -745,9 +745,9 @@ class UpstreamIntegrationTests {
         Edge.sendEvent(experienceEvent) {}
 
         // Verify
-        // Network response assertions
         realNetworkService.assertAllNetworkRequestExpectations()
-        val matchingResponses = realNetworkService.getResponsesFor(networkRequest = invalidNetworkRequest)
+
+        val matchingResponses = realNetworkService.getResponsesFor(invalidNetworkRequest)
 
         Assert.assertEquals(1, matchingResponses?.size)
```

Comment:
Follow similar usages everywhere. 

Import all the required functions like the configOverrideIntegrationTests and update the usages.
```
import org.junit.Assert.assertEquals
import org.junit.Assert.assertNull
```